scripts/misc/generate_variation_set_table.pl

```diff
@@ -22,7 +22,7 @@ =head1 CONTACT
   developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
 
   Questions may also be sent to the Ensembl help desk at
-  <http://www.ensembl.org/Help/Contact>.
+  <helpdesk.org>.
```

Comment:
This doesn't look right; should it not be http://www.ensembl.org/Help/Contact?

Reply:
Strange URL indeed! 
FIxed in the next commit.
Thanks!